### PR TITLE
add swift:5.3 kind and change default from swift:4.2 to swift:5.3

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

**Review by rabbah — APPROVED** [review]

File: ansible/files/runtimes.json [hunk 1]
```diff
@@ -128,6 +128,20 @@
                     "attachmentName": "codefile",
                     "attachmentType": "text/plain"
                 }
+            },
+            {
+                "kind": "swift:5.3",
+                "default": false,
```
Comment (rabbah) [inline ansible/files/runtimes.json:134]: intended to leave as not the default?

Comment (dgrove-oss) [reply ansible/files/runtimes.json:134]: wasn't sure what the right thing to do was.  Jumping from 4.2 to 5.3 as the default seemed aggressive, but maybe we should.

Comment (rabbah) [reply ansible/files/runtimes.json:134]: I defer to your judgement.

Comment (dgrove-oss) [reply ansible/files/runtimes.json]: decided to go for making 5.3 as default.  Will update PR title.

**Review by rabbah — APPROVED** [review]

**Pull request merged**